src/Screens/Main/Chat/Title.tsx

```diff
@@ -19,12 +23,32 @@ type OwnProps = {
   style?: RN.StyleProp<RN.ViewStyle>;
   onPress: () => void | undefined;
   buddyId: string;
+  isBanned?: boolean;
```

Comment:
wouldn't it be more clear if you either are or are not banned and not undefined | false | true : D

Reply:
Aah this is reused? Then nothing

Reply:
is Banned should come from state as we have buddyId?